web/gui/dashboard.js

```diff
@@ -4194,23 +4194,6 @@ NETDATA.peityChartCreate = function (state, data) {
     return true;
 };
 
-// ----------------------------------------------------------------------------------------------------------------
-// "Text-only" chart - Just renders the raw value to the DOM
-
-NETDATA.textOnlyCreate = function(state, data) {
-    var decimalPlaces = NETDATA.dataAttribute(state.element, 'textonly-decimal-places', 1);
-    var prefix = NETDATA.dataAttribute(state.element, 'textonly-prefix', '');
-    var suffix = NETDATA.dataAttribute(state.element, 'textonly-suffix', '');
-
-    // Round based on number of decimal places to show
-    var precision = Math.pow(10, decimalPlaces);
-    var value = Math.round(data.result[0] * precision) / precision;
-
-    state.element.textContent = prefix + value + suffix;
-    return true;
-}
-
-NETDATA.textOnlyUpdate = NETDATA.textOnlyCreate;
```

Comment:
why these changes?

Reply:
Honestly, no clue. Reset to master in 03ba0d113bcc72f6b0e3f3e56625a6ee1a6501e8.